manual.asc

```diff
@@ -231,6 +231,7 @@ Enable / Disable Whiteboard :: This action enables or disables the whiteboard fe
 which is particularly useful for practicing drawing characters from languages such as Japanese. When the whiteboard has been enabled for the current deck, 
 two new actions will become available for clearing and hiding the whiteboard. Disabling the whiteboard will hide these actions as well as the whiteboard itself.
 Deck options :: Open the deck specific study options. See the http://ankisrs.net/docs/manual.html#deckoptions[desktop documentation] for more information about these study options.
+Check Pronunciation :: This action enables or disables the temporary audio recorder tool bar at the top of the card. This feature allows you to record your voice and replay it. It is useful to self checks your own pronunciation. This tool bar is composed of three buttons to : play, stop playing and record audio from microphone. This tool can be used at any time either during question side or than answer side.
```

Comment:
A few minor changes. To note: @mikehardy is much better at writing copy than myself, so there may be a second suggested edit.
```suggestion
Check Pronunciation :: This action enables or disables the temporary audio recorder toolbar at the top of the card. This feature allows you to record your voice and replay it. It is used primarily to check your pronunciation. This toolbar is composed of three buttons: play, stop playing and record microphone audio. This tool can be used while viewing either the question or the answer.
```

Reply:
I like those suggestions - and most of my copy writing "skill" is only evident when I super-focus on the goal of "deleting half the words" combined with "explain it to my mother". Seriously that's my only trick haha